docker-compose.yml

```diff
@@ -27,13 +27,18 @@ services:
 #    command: sleep infinity
 
   db:
-    image: bitnami/mariadb
+    image: mysql:8
     environment:
-      - ALLOW_EMPTY_PASSWORD=yes
-      - MARIADB_DATABASE=archivesspace
-      - MARIADB_USER=as
-      - MARIADB_PASSWORD=as123
-      - TZ=America/Detroit
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+      - MYSQL_ROOT_PASSWORD=asroot123
+      - MYSQL_DATABASE=archivesspace
+      - MYSQL_USER=as
+      - MYSQL_PASSWORD=as123
+    command: [
+      '--character-set-server=utf8mb4',
+      '--collation-server=utf8mb4_unicode_ci',
+      '--log_bin_trust_function_creators=1'
```

Comment:
When I didn't use this flag, one of the migrations failed with something about binary logs and the user not having `SUPER` privileges. They suggest the possibility of using this command, and that seems to be what ArchivesSpace itself is doing for local development.

Reply:
Yes.  I vaguely remember running into binary logs while working on this project so you may come across that same setting in an environment variable of a deployment yaml.